pipeline.ml

```diff
@@ -12,118 +12,144 @@ let read_fpath p = Bos.OS.File.read p |> Rresult.R.error_msg_to_invalid_arg
 let read_channel_uri p =
   read_fpath p |> String.trim |> Uri.of_string |> Current_slack.channel
 
+let pool = Current.Pool.create ~label:"docker" 1
+
+let get_commit ~head =
+  let { Github.Repo_id.owner; name } = Github.Api.Commit.repo_id head in
+  let hash = Github.Api.Commit.hash head in
+  let res = hash ^ owner ^ name in
+  res
```

Comment:
Same as just having `hash ^ owner ^ name` as the last line, but fine if you're deliberately naming it.